example/src/main/java/org/apache/rocketmq/example/benchmark/TransactionProducer.java

```diff
@@ -153,35 +150,37 @@ private static Message buildMessage(final int messageSize) throws UnsupportedEnc
     }
 }
 
-class TransactionExecuterBImpl implements LocalTransactionExecuter {
-
-    private boolean ischeck;
-
-    public TransactionExecuterBImpl(boolean ischeck) {
-        this.ischeck = ischeck;
-    }
-
-    @Override
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg) {
-        if (ischeck) {
-            return LocalTransactionState.UNKNOW;
-        }
-        return LocalTransactionState.COMMIT_MESSAGE;
-    }
-}
-
-class TransactionCheckListenerBImpl implements TransactionCheckListener {
+//class TransactionExecuterBImpl implements LocalTransactionExecuter {
+//
+//    private boolean ischeckLocal;
+//
+//    public TransactionExecuterBImpl(boolean ischeck) {
+//        this.ischeck = ischeck;
+//    }
+//
+//    @Override
+//    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg) {
+//        if (ischeck) {
+//            return LocalTransactionState.UNKNOW;
+//        }
+//        return LocalTransactionState.COMMIT_MESSAGE;
+//    }
+//}
+
+class TransactionListenerImpl implements TransactionListener {
     private boolean ischeckffalse;
     private StatsBenchmarkTProducer statsBenchmarkTProducer;
+    private boolean ischeckLocal;
 
-    public TransactionCheckListenerBImpl(boolean ischeckffalse,
-        StatsBenchmarkTProducer statsBenchmarkTProducer) {
+    public TransactionListenerImpl(boolean ischeckffalse, boolean isCheckLocal,
```

Comment:
ischeckffalse ？

Reply:
Sorry, the last commit did not modify the original code, including variable naming, but the latest commit has been changed.